pages/emailSender.tsx

```diff
@@ -435,15 +486,49 @@ const EmailSender: NextPage = () => {
               ))}
             </SectionContainer>
           </FormControl>
+          <SectionContainer>
+            <StyledSubHeader variant="h5">
+              4) Upload and import csv
+            </StyledSubHeader>
+            <input
```

Comment:
Maybe the attachment button should be in step 2 with the email content text box in the UI